src/USER-CONP/compute_conp_matrix.cpp

```diff
@@ -563,55 +519,32 @@ void ComputeConpMatrix::matrix_assignment() {
       if (mat2tag[ii] == tag[i]) mpos[i] = ii;
 
   memory->destroy(igroupnum_list);
-  memory->destroy(jgroupnum_list);
   memory->destroy(idispls);
-  memory->destroy(jdispls);
   memory->destroy(itaglist_local);
-  memory->destroy(jtaglist_local);
   memory->destroy(itaglist);
-  memory->destroy(jtaglist);
 }
 
 /* ---------------------------------------------------------------------- */
 
 void ComputeConpMatrix::allocate() {
   memory->create(mat2tag, ngroup, "coul/matrix:mat2tag");
-  gradQ_V_inv = new double[ngroup * ngroup];
   gradQ_V = new double *[ngroup];
-  for (bigint i = 0; i < ngroup; i++) gradQ_V[i] = new double[ngroup];
+  array = new double *[ngroup];
```

Comment:
Wozu benutzt du genau array? Der Name ist etwas nichtsagend oder muss das so sein?

Reply:
Das ist muss so, weil es in public von compute.h ist. D.h. es einfach vom fix darauf zuzugreifen.